src/components/button/button.ts

```diff
@@ -255,7 +258,7 @@ export default class SlButton extends ShoelaceElement implements ShoelaceFormCon
         })}
         ?disabled=${ifDefined(isLink ? undefined : this.disabled)}
         type=${ifDefined(isLink ? undefined : this.type)}
-        title=${'' /* An empty title prevents browser validation tooltips from appearing on hover */}
+        title=${this.title}
```

Comment:
Can we preserve the comment? This will let folks know _why_ we're setting a global attribute and help prevent regressions.

Reply:
Yeah, you're right. I changed it.